src/source-dpdk.c

```diff
@@ -375,8 +412,21 @@ static TmEcode ReceiveDPDKLoop(ThreadVars *tv, void *data, void *slot)
     TmThreadsSetFlag(tv, THV_RUNNING);
     PacketPoolWait();
 
+    if (ptv->intr_en) {
+        if (InterruptsRXEnable(ptv->port_id, ptv->queue_id) == 0) {
+            SCLogDebug("Enabling interrupt for port %d queue %d", ptv->port_id, ptv->queue_id);
+            ptv->intr_en = true;
+        } else {
+            SCLogConfig(
```

Comment:
should this be a warning or error? It's clear that the user isn't getting what they want

Reply:
yeah lets go with error

Reply:
errorring